dune-project

```diff
@@ -1,14 +1,13 @@
-(lang dune 2.8)
+(lang dune 2.9)
 (name js_of_ocaml)
 
 (generate_opam_files true)
 
 (authors "Ocsigen team <dev@ocsigen.org>")
 (maintainers "Ocsigen team <dev@ocsigen.org>")
 (source (github ocsigen/js_of_ocaml))
-(bug_reports "https://github.com/ocsigen/js_of_ocaml/issues")
```

Comment:
Why is bug_reports gone ?

Reply:
If github is specified in the source stanza, it's generated automatically.